core/common/src/LocalDateTime.kt

```diff
@@ -13,29 +13,29 @@ import kotlinx.serialization.Serializable
 /**
  * The representation of a specific civil date and time without a reference to a particular time zone.
  *
- * This class does not describe specific *moments in time*, which are represented as [Instant] values.
- * Instead, its instances can be thought of as clock readings, something that someone could observe in their time zone.
- * For example, `2020-08-30T18:43` is not a *moment in time*, since someone in Berlin and someone in Tokyo would witness
+ * This class does not describe specific *moments in time* represented as [Instant] values.
```

Comment:
The idea: "This class does not describe specific moments in time. `Instant` is the thing that represents moments in time; if you reach for `LocalDateTime` for this use case, don't; use `Instant` instead". I think the proposed version doesn't convey that.

Reply:
Thank you, what about "This class does not describe specific *moments* in time. For that, use [Instant] values instead."